doc/source/user_guide/computation.rst

```diff
@@ -334,6 +339,45 @@ compute the mean absolute deviation on a rolling basis:
    @savefig rolling_apply_ex.png
    s.rolling(window=60).apply(mad, raw=True).plot(style='k')
 
+Additionally, :meth:`~Rolling.apply` can leverage `Numba <https://numba.pydata.org/>`__
+if installed as an optional dependency as the execution engine of the apply aggregation using the
+``engine='numba'`` and ``engine_kwargs`` arguments (``raw`` must also be set to ``True``).
+Numba will be applied in potentially two routines:
+
+1. If ``func`` is a standard Python function, the engine will JIT the passed function. ``func``
+can also be a pre-JIT function in which case the engine will not JIT the function again.
```

Comment:
is pre-JIT a term? I would say if its just a JITTED function its ok; can you add a numba refernce to what a JIT function is